README.md

```diff
@@ -0,0 +1,31 @@
+# device register
+
+`device-register` registers a device on the cacophony server so a device can upload thermal or audio recordings to the server
+
+Project | device-register
```

Comment:
The title row is redundant and confusing because these items aren't actually titles for the remainder of the table.